### ISSUE TITLE
na_ontap_user module displays 'Module did not set no_log for set_password'

### ISSUE THREAD

**Issue by ecantin** [ansible/ansible#63236]
##### SUMMARY

1) On all 'na_ontap_user' invocations by a task, the following warning message is displayed:
 [WARNING]: Module did not set no_log for set_password

2) On all 'na_ontap_user' invocation by a task with '-vvv' ansible-playbook option, the 'set_password' option value is displayed in clear text.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

na_ontap_user

##### ANSIBLE VERSION

```
$ ansible --version
ansible 2.7.2
  config file = /home/ansible/.tmp/dev/test_hw/2018-11-23_00_netapp/ansible.cfg
  configured module search path = [u'/home/ansible/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, May  3 2017, 07:55:04) [GCC 4.8.5 20150623 (Red Hat 4.8.5-14)]
```

##### CONFIGURATION

```
$ ansible-config dump --only-changed
DEFAULT_HOST_LIST(/home/ansible/.tmp/dev/test_hw/2018-11-23_00_netapp/ansible.cfg) = [u'/home/ansible/.tmp/dev/test_hw/2
DEFAULT_STDOUT_CALLBACK(/home/ansible/.tmp/dev/test_hw/2018-11-23_00_netapp/ansible.cfg) = yaml
```

##### OS / ENVIRONMENT

$ uname -a; cat /etc/redhat-release
Linux vel1-nfv02-cnode01 3.10.0-693.21.1.el7.x86_64 #1 SMP Fri Feb 23 18:54:16 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
Red Hat Enterprise Linux Server release 7.4 (Maipo)

##### STEPS TO REPRODUCE

Run a playbook which includes a task invoking 'na_ontap_user' module with 'set_password' option.

```
$ cat netapp_0013506.yml
---
- name: NetApp
  hosts: netapp99
  connection: local
  gather_facts: false
  vars:
    netappSvmName: svm99
    login: &login
      hostname: "{{ ansible_host }}"
      username: "{{ netappUsername }}"
      password: "{{ netappPassword }}"
      https: true
      validate_certs: false
    netappState: present

  tasks:
    - name: SVM
      na_ontap_svm:
        name: "{{ netappSvmName }}"
        allowed_protocols: ["nfs"]
        language: "fr.utf_8"
        root_volume: svm99_vol_root
        root_volume_security_style: unix
        root_volume_aggregate: aggr1_stor02
        snapshot_policy: "none"
        subtype: "default"
        <<: *login

    - name: SVM user
      na_ontap_user:
        vserver: "{{ netappSvmName }}"
        name: user99
        application: ontapi
        authentication_method: password
        set_password: Quae5ieVae1eiCh4io1epah6
        lock_user: no
        role_name: vsadmin-readonly
        <<: *login
```

##### EXPECTED RESULTS

No warning message.
No display of the 'set_password' option value with '-vvv' ansible-playbook option.

##### ACTUAL RESULTS

```
$ ansible-playbook netapp_0013506.yml -i inventory_0013506

PLAY [NetApp] **********************************************************************************************************

TASK [SVM] *************************************************************************************************************
changed: [netapp99]

TASK [SVM user] ********************************************************************************************************
 [WARNING]: Module did not set no_log for set_password

changed: [netapp99]

PLAY RECAP *************************************************************************************************************
netapp99                   : ok=2    changed=2    unreachable=0    failed=0

$ ansible-playbook netapp_0013506.yml -i inventory_0013506 -vvv |& grep set_password
 [WARNING]: Module did not set no_log for set_password
      set_password: Quae5ieVae1eiCh4io1epah6
```

**Comment by carchi8py** [comment]
This was fixed in ansible 2.8

**Comment by ecantin** [comment]
Validation with ansible v2.9.4: success, no more warning.
